### PR TITLE
Unused deps: remove them

### PR DESCRIPTION
**Pull request by danielbeardsley**
I've noticed some unmet peer dependencies and some unused dependencies. This change drops some that seem like they aren't needed.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -110,7 +110,6 @@
       "eslint-plugin-react": "7.23.0",
       "eslint-plugin-react-hooks": "4.2.0",
       "graphql-tag": "2.12.6",
-      "graphqurl": "1.0.1",
```
Comment (batbattur) [inline frontend/package.json:115]: It looks like we use the cli `gq` version here:
https://github.com/iFixit/react-commerce/blob/d528e7820d58d4f6f1ea3fd6e1f14e40ebf42f2b/frontend/lib/shopify-storefront-sdk/generate-schema.js#L20

Found it by running `git grep "gq "`

**Comment by batbattur** [comment]
dev_block 🚛 on double checking `graphqurl`, ref https://github.com/iFixit/react-commerce/pull/1256#discussion_r1070150062.

File: frontend/package.json (hunk 1 above)
Comment (danielbeardsley) [reply frontend/package.json:115]: OMG, thank you... this is the the second time I've done this exact thing. Derp. Will fix.

**Comment by danielbeardsley** [comment]
un_dev_block :+1: 

Thanks @batbattur !

File: frontend/package.json [hunk 2]
```diff
@@ -129,5 +130,12 @@
    },
    "msw": {
       "workerDirectory": "public"
+   },
+   "pnpm": {
+      "peerDependencyRules": {
+         "allowedVersions": {
+            "graphql": "15"
+         }
+      }
```
Comment (masonmcelvain) [inline frontend/package.json:139]: I remember rush complains about unmet peer deps, as part of QA we should check that this doesn't break updating the submodule in our php app

**Review by masonmcelvain — APPROVED** [review]
CR ✌🏻 Nice!

File: frontend/package.json (hunk 2 above)
Comment (danielbeardsley) [reply frontend/package.json:139]: Fair... but this pull *reduces* the number of unmet peer deps, not increases.

Comment (masonmcelvain) [reply frontend/package.json:139]: Oops I didn't read the docs, I thought this config was doing something different. Seems like there's no concern here.

> Unmet peer dependency warnings will not be printed for peer dependencies of the specified range.

https://pnpm.io/package_json#pnpmpeerdependencyrulesallowedversions

**Comment by k0rvusk0r4x** [comment]
Smoke test shows everything continues to work the same as master.

QA 😺

**Pull request merged**